src/Controls/src/Core/Handlers/Shell/Windows/ShellFlyoutItemView.cs

```diff
@@ -117,7 +117,7 @@ void ShellElementPropertyChanged(object sender, PropertyChangedEventArgs e)
 						fe.Visibility = WVisibility.Visible;
 					}
 				}
-
+				base.MeasureOverride(availableSize);
```

Comment:
@Foda This reminds me of something you were looking at recently?

Reply:
Yeah, similar to https://github.com/dotnet/maui/pull/25479

I'm curious if we should just have remove the measure override. I don't really see the point of it